Sources/Workspace/Workspace.swift

```diff
@@ -853,8 +853,9 @@ extension Workspace {
         forBinaryArtifactAt path: AbsolutePath,
         diagnostics: DiagnosticsEngine
     ) -> String {
-        guard fileSystem.isFile(path) else {
-            diagnostics.emit(error: "file not found at path: \(path.pathString)")
+        guard let pathExtension = path.extension, archiver.supportedExtensions.contains(pathExtension) else {
```

Comment:
We should still check that the path is a file. Directories can have extensions.

Reply:
Thanks! Re-added, also makes sense the path may not exist on disk.